test/seccomp_oci_artifacts.bats

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bats
+# vim:set ft=bash :
+
+load helpers
+
+function setup() {
+	if ! "$CHECKSECCOMP_BINARY"; then
+		skip "seccomp is not enabled"
+	fi
+
+	setup_test
+}
+
+function teardown() {
+	cleanup_test
+}
+
+ARTIFACT_IMAGE_WITH_ANNOTATION=quay.io/saschagrunert/nginx:seccomp
+ARTIFACT_IMAGE=quay.io/saschagrunert/seccomp:v1
+ANNOTATION=io.kubernetes.cri-o.seccompProfile
+
```

Comment:
can you add a test for if the artifact annotation is bogus?

Reply:
Done